eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -182,39 +195,37 @@ void findsProbesWithVersionsInDifferentObjectionaries(@TempDir final Path temp)
             .with("hsh", first)
             .withVersionedProgram()
             .execute(new FakeMaven.Probe());
+        final ObjectName text = new OnVersioned("org.eolang.txt.text", "5f82cc1");
         MatcherAssert.assertThat(
             String.format(
-                "Tojos should have contained versioned object %s after probing, but they didn't",
-                ProbeMojoTest.STDOUT
-            ),
-            maven.externalTojos().contains(ProbeMojoTest.STDOUT),
-            Matchers.is(true)
-        );
-        MatcherAssert.assertThat(
-            String.format(
-                "Tojos should have contained versioned object %s after probing, but they didn't",
-                text
+                "Tojos should contain versioned objects '%s' after probing, but they didn't",
+                Arrays.asList(text, ProbeMojoTest.STDOUT)
             ),
-            maven.externalTojos().contains(text),
+            maven.externalTojos().contains(text, ProbeMojoTest.STDOUT),
             Matchers.is(true)
         );
         MatcherAssert.assertThat(
-            "First entry of tojos after probing should have contained two probed objects, but it didn't",
+            "Program tojo after probing should contain exactly two probed objects",
             maven.programExternalTojo().probed(),
             Matchers.equalTo("2")
         );
         MatcherAssert.assertThat(
-            "First entry of tojos after probing should have contained given hash, but it didn't",
+            "Program tojo after after probing should have given hash",
             maven.programExternalTojo().hash(),
             Matchers.equalTo(first.value())
         );
     }
 
-    private static String program() {
-        return new UncheckedText(
-            new TextOf(
-                new ResourceOf("org/eolang/maven/simple-io.eo")
-            )
-        ).asString();
+    private static String[] program() {
```

Comment:
@volodya-lombrozo I think it would be better to add javadoc here

Reply:
@maxonfjvipon I believe if `qulice` doesn't tell about it, it could be avoided.